src/scene/scene_phongmaterial.js

```diff
@@ -210,6 +210,7 @@ pc.extend(pc, function () {
             this.specularMapTint = false;
             this.emissiveMapTint = false;
             this.emissiveIntensity = 1;
+            this.mask = 1;

```

Comment:
Yep. But right now, if you wrote another material based on a shader generator, you'd need this in the pc.Material base class. Seems like it belongs there to me.